src/pfe/file-watcher/server/test/functional-test/suites/generic/tests/project.ts

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+import mocha from "mocha";
+import { expect } from "chai";
+import * as _ from "lodash";
+import path from "path";
+
+import * as genericLib from "../../../lib/generic";
+
+import * as app_configs from "../../../configs/app.config";
+
+export default class ProjectTest {
+    testName: string;
+
+    constructor(testName: string) {
+        this.testName = testName;
+    }
+
+    run(runOnly?: boolean): void {
+        (runOnly ? describe.only : describe)(this.testName, () => {
+            this.runGetProjectTypes();
+        });
+    }
+
+    private runGetProjectTypes(): void {
+        describe("getProjectTypes function", () => {
+            it("get project type without a location", async () => {
+                const info: any = await genericLib.getProjectTypes();
```

Comment:
We don't have the PFE log for the test case `get project type without a location`, but the test cases passed then we should already verified the output from PFE and the PFE should have the output in the PFE log. If the test cases failed and we don't have PFE log, then it's not good for us to debug. Wonder is this due to the issue of combining PFE log and Mocha tests log?

![Screen Shot 2019-08-12 at 2 35 21 PM](https://user-images.githubusercontent.com/15131537/62888990-6f89bb80-bd0e-11e9-91bc-c1ade467f040.png)

Reply:
I think it doesn't output the log because by default the log level is set to `INFO` and with Rajiv's change on logs, it will not print trace or debug level messages such as this case.

Reply:
If thats the case, should the tests run with default log level of "debug"

Reply:
Yeah, if only `DEBUG` log level can print that log message, we need to to either set default log level to `DEBUG` or set to `DEBUG` then set it back to `INFO`

Reply:
Yes, I can set it to `DEBUG` by default